Chummer/data/vehicles.xml

```diff
@@ -10203,6 +10203,236 @@
         </gear>
       </gears>
     </vehicle>
+    <vehicle>
+      <id>be13725c-d5cd-4855-96f6-a07de090b58d</id>
+      <name>Eurocar Escape FüKomKW</name>
```

Comment:
"FüKomKW" I assume should be translated to "Command Truck", "Command and Control Station", "Command Center", or something similar.

Reply:
I dont think that it makes sense to translate the unique german abbrevations